index.html

```diff
@@ -243,6 +243,32 @@ <h1>
                 </div>
             </div>
         </section>
+        <section class="contact-me">
+
+            <div class="contact-text">
+                <h2>Contact me</h2>
+
+                <p>If you have an application you are interested in developing, a feature that you need built
+                    or a project that you need coding. I'd love to help with it.</p>
+            </div>
+
+            <div class="form">
+                <form action="https://formspree.io/f/xoqrdelo" method="post">
+                    <ul>
+                        <li><input type="text" name="name" placeholder="Your Name" maxlength="30" required></li>
+                        <li><input type="email" name="email" placeholder="Your Email" required></li>
+                        <li><textarea name="message" placeholder="Write your message here" maxlength="500"
+                                required></textarea></li>
+                        <li><button class="button-submit" type="submit">Get It Touch</button></li>
```

Comment:
- [x] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your submit button should be aligned to the left, Right now it is in the center. Please align your button to the left and make sure it matches the design.

Your design

![Screenshot (101)](https://user-images.githubusercontent.com/83803414/154106974-9d034e80-8885-4610-9d91-ea428ae6e3e6.png)


Figma design

![Screenshot (103)](https://user-images.githubusercontent.com/83803414/154106957-38656734-f021-4c64-8fd9-bbe245138c4c.png)